vep/src/main/scala/VepStage.scala

```diff
@@ -23,11 +23,6 @@ class VepStage(implicit context: Context) extends Stage {
 
   val variants: Input.Source = Input.Source.Success("out/varianteffect/variants/")
 
-  /** Additional resources that need uploaded to S3. */
-  override def additionalResources: Seq[String] = Seq(
-    "runVEP.sh"
-  )
-
   /** Input sources. */
```

Comment:
So runVEP.sh will still get onto the EMR nodes just by virtue of being in the resources directory?  Does that mean additionalResources should only be used when there's a requirement for something outside of the resources dir?

Reply:
additional resources seems to be something that was either added as a tack on or for testing or something, it is rarely used. But I will probably start using it more in cases where it makes sense to split out functionality that will be shared among a bunch of different jobs in the same project.

It was originally there I think because it used to use the pl script (which called the sh script). But then that changed and someone (including me) forgot to remove it. Is ends up in the cluster by virtue of being the run script in the make function. Most of the time jobs just pull in (unchanging) utility functions from the bin folder. This is just a way to pull in something which is slightly more likely to change run to run or to be modified in the future where bin would be a nuisance I think.